tasks/application-astro.tex

```diff
@@ -18,10 +18,18 @@
   the standardized protocols defined by the International Virtual Observatory
   Alliance.
 
+\begin{figure}[ht!]\centering
+  \includegraphics[width=0.6\textwidth]{python-astro-citations}
+  \caption{Mentions of programming languages in refereed Astronomy papers, extracted from ADS. Python usage has increased dramatically in the recent years.}\label{fig:python-astro-citations}
+\end{figure}
+
   Python and notebooks are rapidly increasing in importance for astronomy 
   research. Indeed, Python for Astronomy software ecosystem has known a 
   constant steady growth in the latest years, as shown in 
-  figure~\ref{fig:python-astro-citations}.
+  figure~\ref{fig:python-astro-citations}. As Python and notebooks integrate 
+  well together, the Jupyter notebook as an analysis tool is becoming a hot 
+  topic in the astronomical world: large surveys like the LSST (Large Synoptic
+  Survey Telescope) have endorsed the usage of the Jupyter platform.\\
```

Comment:
That's perfect. Maybe add a reference to the LSST Jupyter portal? I think I saw a talk about this at JupyterCon 2018. No worries if you can't find a reference - not that important.